src/utilities/network/lookupApi.js

```diff
@@ -6,7 +6,10 @@
 import { SETTINGS_KEYS } from '../../settings';
 import { UIDatabase } from '../../database';
 
-const { SYNC_URL } = SETTINGS_KEYS;
+import { NAME_TYPES } from '../../sync/syncTranslators';
```

Comment:
an issue for a `sync/constants` file what you reckon

Reply:
Agreed! Add sync constants is on the patient API epic issues list (no issue created yet, but will do once I work my way to it!).